pyproject.toml

```diff
@@ -1,9 +1,124 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs", "nipreps-versions"]
+build-backend = "hatchling.build"
+
+[project]
+name = "nibabies"
+description = "Processing workflows for magnetic resonance images of the brain in infants"
+readme = "long_description.md"
+authors = [{name = "The NiPreps Developers", email = "nipreps@gmail.com"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
+dependencies = [
+    "migas >= 0.3.0",
+    "nibabel >= 5.0.0",
+    "nipype >= 1.8.1",
+    "nitime",
+    "nitransforms >= 21.0.0",
+    "niworkflows @ git+https://github.com/nipreps/niworkflows.git@master",
+    "numpy >= 1.21.0",
+    "packaging",
+    "pandas",
+    "psutil >= 5.4",
+    "pybids >= 0.15.0",
+    "requests",
+    "sdcflows @ git+https://github.com/nipreps/sdcflows.git@master",
+    "smriprep @ git+https://github.com/nipreps/smriprep.git@master",
+    "tedana ~= 0.0.12",
+    "templateflow >= 0.6",
+    "toml",
+]
+dynamic = ["version"]
+
+[project.urls]
+Documentation = "https://nibabies.readthedocs.io/en/latest/"
+"Source Code" = "https://github.com/nipreps/nibabies"
+"Bug Tracker" = "https://github.com/nipreps/nibabies/issues"
+"Docker Images" = "https://hub.docker.com/r/nipreps/nibabies"
+
+[project.optional-dependencies]
+dev = [
+    "black ~= 22.3.0",
+    "isort ~= 5.10.1",
+    "pre-commit",
+]
+doc = [
+    "pydot >= 1.2.3",
+    "sphinx >= 1.8",
+    "sphinx-argparse",
+    "sphinx_rtd_theme",
+    "myst_nb",
+]
+duecredit = ["duecredit"]
+maint = [
+    "fuzzywuzzy",
+    "python-Levenshtein",
+]
+test = [
+    "coverage",
+    "codecov",
+    "pytest",
+    "pytest-cov",
+    "pytest-env",
+]
+# Aliases
+docs = ["nibabies[doc]"]
+tests = ["nibabies[test]"]
+all = ["nibabies[dev,doc,maint,telemetry,test]"]
+
+[project.scripts]
+nibabies = "nibabies.cli.run:main"
+
+[tool.hatch.metadata]
+allow-direct-references = true
+
+[tool.hatch.build.targets.sdist]
+exclude = [".git_archival.txt"]  # No longer needed in sdist
+
+[tool.hatch.build.targets.wheel]
+packages = ["nibabies"]
+
+[tool.hatch.version]
+source = "vcs"
+raw-options = { version_scheme = "nipreps-calver" }
+
+[tool.hatch.build.hooks.vcs]
+version-file = "nibabies/_version.py"
+
+[tool.distutils.bdist_wheel]
+universal = true
```

Comment:
We don't support Python 2 except in the wrapper. I don't think you want this here.

```suggestion
```

Reply:
I was following https://github.com/nipreps/fmriprep/commit/c4acb74ba77c5e7f6eea6649d31e485c9cb84162

Reply:
That looks like an error in my PR.